api/src/test/java/com/epam/pipeline/test/acl/AclTestBeans.java

```diff
@@ -484,6 +479,14 @@ protected FolderCrudManager mockFolderCrudManager() {
         return Mockito.mock(FolderCrudManager.class);
     }
 
+    @Bean
```

Comment:
Also, could I ask you to add not issue related changes to this class?
If you don't mind, let's refactor beans above `JsonService`, `DataStorageManager` and `FolderCrudManager` to make them `@MockBean`. Moreover, let's remove `TemplatesScanner` since it is not a bean.